bucket/android-clt.json

```diff
@@ -6,26 +6,51 @@
         "identifier": "Freeware",
         "url": "https://developer.android.com/studio/terms.html"
     },
-    "notes": "You need to launch sdkmanager with --sdk_root=$env:ANDROID_SDK_ROOT.",
     "url": "https://dl.google.com/android/repository/commandlinetools-win-9477386_latest.zip",
     "hash": "696431978daadd33a28841320659835ba8db8080a535b8f35e9e60701ab8b491",
+    "suggest": {
+        "Java": "java/openjdk17"
+    },
+    "extract_to": "cmdline-tools",
+    "pre_install": "Rename-Item \"$dir\\cmdline-tools\\cmdline-tools\" -NewName 'latest'",
     "post_install": [
-        "if (Test-Path \"$(appdir adb $global)\") {",
-        "    New-Item \"$dir\\platform-tools\" -ItemType Junction -Target \"$(appdir adb $global)\\current\\platform-tools\" | Out-Null",
+        "# Persistence",
+        "Get-ChildItem \"$persist_dir\\cmdline-tools\" -Exclude 'latest' | Move-Item -Destination \"$dir\\cmdline-tools\" -Force",
+        "@('emulator', 'ndk-bundle', 'platform-tools', 'tools') | ForEach-Object {",
+        "  if (Test-Path \"$persist_dir\\$_\") {",
+        "    Move-Item \"$persist_dir\\$_\" \"$dir\" -Force",
+        "  }",
+        "}",
```

Comment:
I am not sure if these files will be lost due to unexpected errors, but they will take up a lot of disk space when many SDK packs was downloaded.